python/hail/keytable.py

```diff
@@ -1394,29 +1394,31 @@ def maximal_independent_set(self, i, j):
         kinship.
 
         >>> related_pairs = hc.import_vcf("data/sample.vcf.bgz").pc_relate(2, 0.001).filter("kin > 0.125")
-        >>> related_samples = high_kin.query('i.flatMap(i => [i,j]).collectAsSet()')
-        >>> related_samples_to_keep = high_kin.maximal_independent_set("i", "j")
-        >>> related_samples_to_remove = high_kin_samples - set(independent_set_list)
-        >>> vds.filter_samples_list(list(samples_to_remove))
+        >>> related_samples = related_pairs.query('i.flatMap(i => [i,j]).collectAsSet()')
+        >>> related_samples_to_keep = related_pairs.maximal_independent_set("i", "j")
+        >>> related_samples_to_remove = related_samples - set(related_samples_to_keep)
+        >>> vds.filter_samples_list(list(related_samples_to_remove))
 
         **Notes**
 
-        Each row of the :class:`.KeyTable` corresponds to an undirected edge
-        between the nodes given by ``i`` and ``j``. An undirected edge may
-        appear multiple times in the :class:`.KeyTable`. It will not affect the
-        output. The node set of the graph is implicitly all the possible values
-        of ``i`` and ``j`` in this :class:`.KeyTable`.
+        Each row of the key table corresponds to an undirected edge between the
+        vertices given by evaluating ``i`` and ``j`` on that row. An undirected
+        edge may appear multiple times in the key table and will not affect the
+        output. The vertex set of the graph is implicitly all the possible
```

Comment:
I'd remove "possible". It's unnecessarily abstract to consider an infinite graph when there are finitely many rows.

Reply:
And I still think this should be the first sentence.